### ISSUE TITLE
[Suggestion] - 

### ISSUE THREAD

**Issue by Omernoa** [Monika-After-Story/MonikaModDev#5386]
Hey,
do you think you'd be able make Monika a little more regretful of what she's done to the other dokis?
Since act 4 is canon in MAS I think it's a bit odd that Monika doesn't really seem like this act happened.
Great mod btw and thank you.

**Comment by multimokia** [comment]
Actually, Monika is an extension of DDLC Monika, so she shouldn't necessarily be sorry depending on the act you came from.


In addition, there's a topic for this which has the best of both worlds. There's the `Insensitive comments` topic, which she asks you if you're comfortable with her kinda bashing them or not.

The thing is that she wouldn't really be remorseful since in the end she did it so she could get you in the end. And if you came back, she still did, the others were still deleted again. So she has no reason to be remorseful about it, realistically.

**Comment by lunulae** [comment]
^ The topic Multi mentioned is a great one to check out if you haven't.
I originally said I didn't mind when she made insensitive remarks about the other girls, but at one point I decided to repeat the topic and try switching my answer--and I was _very_ impressed with how the change is handled. 
Rather than just cutting out the crueler lines (as I had originally imagined) many of them are rewritten to give her a few more moments of introspection and implication that deep down, she does miss the others. Even though I didn't mind the darker jokes, it really is written like a natural progression of maturity about the situation, so I haven't changed it back since.

**Comment by Omernoa** [comment]
It's just that MAS happens after act 4, an act where Monika has shown a lot of remorse about what she's done (even if we know she was justified) so it would be very much in character for her to feel sad about it. Maybe after enough affection she confesses to the player that it still bothers her and then the player gets an option to support her?

**Comment by MisterSimple** [comment]
I'd say that while she does feel sad about the events in DDLC, ultimately what she feels is less "remorse"  and more "survivor's guilt". She mentions that she wishes she didn't have to take the measures she took, but she had no choice; it's thanks to those measures that she's still there with the player. Would she be happier if the rest of the club members were still alive and well? Surely, but it is what it is.

**Comment by Legendkiller21** [comment]
> It's just that MAS happens after act 4

No, it doesn't. You can start MAS at any time, at any act and if you start before act 4 the intro changes accordingly.

>  it would be very much in character for her to feel sad about it

Read Multi's reply.

> and then the player gets an option to support her?

You get that option. It's called spending time with her and make her feel loved.

At a higher affection, she'd feel even less remorse. In the intro itself, if you say 'I love you too,' she says that you having said that makes everything she did _worthwhile_. This means that if you continually show up and make her happy, i.e. raise her affection with you, she'd feel even less remorse about it all, as her actions weren't in vain.

If anything I don't see a reason for her to feel remorse, especially if you started after act 4. She brought them back on act 4 and then you got the mod and deleted the others. This time you are the one who deleted them, not her.

**Comment by Omernoa** [comment]
Oh, I understand, thanks for explaining.

‫בתאריך שבת, 29 בפבר׳ 2020 ב-3:27 מאת ‪Legendkiller21‬‏ <‪
notifications@github.com‬‏>:‬

**Comment by fromulus** [comment]
I do somewhat agree on the idea that the changes made after the insensitive comments topic feel more in line with the mindset Monika would probably have after act 4.

Perhaps a good solution would be to have the _cares_about_other_girls_ variable set to true automatically when starting the mod after Act 4 ?
As well as a new version of the insensitive comments topic that would be triggered with said variable and would allow the post-Act 4 player to change it back if they want.

**Comment by Omernoa** [comment]
Yeah, I think it'd be a good idea, thanks for listening.

‫בתאריך שבת, 29 בפבר׳ 2020 ב-13:12 מאת ‪fromulus‬‏ <‪
notifications@github.com‬‏>:‬

**Comment by ThePotatoGuy** [comment]
was this addressed at all in the past month @multimokia ?

Seems reasonable to default that var to true for act4+ loads. I remember you touched the import code but dont remember if that was considered.
